pkg/handler/repository_parameters.go

```diff
@@ -69,14 +69,21 @@ func (rph *RepositoryParameterHandler) validate(c echo.Context) error {
 	validationResponse := make([]api.RepositoryValidationResponse, repoCount)
 	errors := make([]error, repoCount)
 
+	UUIDList := []string{}
+	for i := 0; i < repoCount; i++ {
+		if validationParams[i].UUID != nil {
+			UUIDList = append(UUIDList, *validationParams[i].UUID)
+		}
+	}
+
 	//Use go routine here to reduce the api call time length.
 	// Each url validation can take seconds to fail in case of a timeout.
 	// So this makes the call roughly take the amount of time as a single timeout at worst case
```

Comment:
```suggestion
	// This makes the call roughly take the amount of time as a single timeout at worst.
```
You should avoid starting a sentence with "So"; so, either make the previous sentence end with a comma and change "So" to "so" or accept the simple change proposed above.

Reply:
This wasn't me, happy to change it though.